src/csl/archive.rs

```diff
@@ -217,7 +217,7 @@ pub enum ArchivedStyle {
     ///
     /// Authors: Patrick O'Brien.
     MaryAnnLiebertVancouver,
-    /// Modern Humanities Research Association 3rd edition (note with bibliography).
+    /// Modern Humanities Research Association 4th edition (note with bibliography).
```

Comment:
Hi, can you please remove the following changes from the PR's diff:

1. All changes under `archive/styles`
2. Changes to `archive.rs`

They are currently automatically changed by hayagriva's tests, which is annoying (we will improve this), but for now we should just remove the diff from the PR until this is improved :)